packages/nimble/inst/CppCode/customizeMakeconf.R

```diff
@@ -0,0 +1,9 @@
+try({
+    packageMakeconfFile <- file.path(R.home('etc'), 'Makeconf')
+    if(file.exists(packageMakeconfFile)) {
+        file.copy(packageMakeconfFile, 'Makeconf')
+        lines <- readLines('Makeconf')
+        lines <- gsub('-g', '', lines)
+        writeLines(lines, con = 'Makeconf')
+    }
+    })
```

Comment:
It will help users file bug reports if we don't silence failures. Consider logging
```r
tryCatch({
   # do stuff
}, error = function(e) {
  warning(paste('Failed to remove debug symbols from nimble:', e), sep = '\n')
})
```